lib/manager/terraform/__fixtures__/1.tf

```diff
@@ -178,6 +178,10 @@ terraform {
       source  = "terraform-providers/docker"
       version = "2.7.2"
     }
+    docker2 = {
+      source  = "docker"
```

Comment:
Are you sure this is valid? As far as I know Terraform 0.13 throws an error on this.

Reply:
The reasoning behind this is this documentation: https://www.terraform.io/docs/configuration/provider-requirements.html#source-addresses   
Specifically the note at the bottom.

I have tested it with this code: 

``` hcl 
terraform {
  required_providers {
    aws = {
      source  = "aws"
      version = "~> 3.0"
    }
    kubernetes = {
      source  = "kubernetes"
      version = "~> 1.0"
    }
  }
}
```
![image](https://user-images.githubusercontent.com/17493763/94991748-50d72b80-0585-11eb-947c-8e2f529cc4d1.png)

Reply:
ok, you've removed `docker` in your real test.

```yaml
terraform {
  required_providers {
    docker = {
      source  = "docker"
      version = "2.7.2"
    }
    helm = {
      source  = "helm"
      version = "1.3.2"
    }
    kubernetes = {
      source  = "kubernetes"
      version = "1.13.2"
    }
  }
  required_version = ">= 0.13"
}
```
![image](https://user-images.githubusercontent.com/1798109/95707825-e5fbb380-0c5a-11eb-9678-09dc1d36ed09.png)

so can you please update the test case to me more real world 🙃

Reply:
I will do that!